### DISCUSSION TITLE
Export to CSV is not working

### DISCUSSION THREAD

**Discussion by Stingray91** [AntlerVC/firetable#1512] [Support / Q&A]
The export to CSV for all fields is not working. It seems that the larger tables just don't export (exports fine for JSON). The popup that says that the download will begin soon pops up but nothing downloads.

**Comment by harinij** [comment]
@Stingray91 is the number of rows in the table >10K?

**Comment by Stingray91** [comment]
Yes there are no more than 500 in there
________________________________

**Comment by harinij** [comment]
Hi @Stingray91 - we weren't able to reproduce this. potentially it could be related to the data seen in the table itself? If you can try for smaller number of columns in the export does it work or narrow it down? 

Moving this to discussion as it seems to be related to specific dataset

**Reply by taggxyz** [reply]
I am facing this issue today. Did you ever find out what caused it?
